Sources/StreamChatUI/CommonViews/CellSeperatorView/CellSeparatorView.swift

```diff
@@ -5,36 +5,22 @@
 import StreamChat
 import UIKit
 
-/// The cell separator reusable view that acts as container of the content view.
+/// The cell separator reusable view that acts as container of the visible part of the separator view.
 public typealias CellSeparatorReusableView = _CellSeparatorReusableView<NoExtraData>
 
-/// The cell separator reusable view that acts as container of the content view.
-open class _CellSeparatorReusableView<ExtraData: ExtraDataTypes>: UICollectionReusableView, UIConfigProvider {
+/// The cell separator reusable view that acts as container of the visible part of the separator view.
+open class _CellSeparatorReusableView<ExtraData: ExtraDataTypes>: _CollectionReusableView, UIConfigProvider {
```

Comment:
Nice one 👍 We probably need to do this to the commands suggestion header reusable view, and the other `CollectionReusableView`'s we are using, so every reusable view follows the same pattern